lib/utils.pm

```diff
@@ -266,7 +266,8 @@ sub assert_gui_app {
 # console font, we need to call systemd-vconsole-setup to workaround
 # that
 sub check_console_font {
-    return if check_var('BACKEND', 'spvm');
+    # cover ipmi case now
+    return if (check_var('BACKEND', 'spvm')) || if (check_var('BACKEND', 'ipmi'));
```

Comment:
the second if is not needed here:
`return if check_var('BACKEND', 'spvm') || check_var('BACKEND', 'ipmi');